components/camel-azure/camel-azure-servicebus/src/main/java/org/apache/camel/component/azure/servicebus/ServiceBusConsumer.java

```diff
@@ -30,31 +30,34 @@
 import org.apache.camel.component.azure.servicebus.client.ServiceBusClientFactory;
 import org.apache.camel.component.azure.servicebus.client.ServiceBusReceiverAsyncClientWrapper;
 import org.apache.camel.component.azure.servicebus.operations.ServiceBusReceiverOperations;
+import org.apache.camel.spi.ExceptionHandler;
 import org.apache.camel.spi.Synchronization;
 import org.apache.camel.support.DefaultConsumer;
 import org.apache.camel.support.SynchronizationAdapter;
 import org.apache.camel.util.ObjectHelper;
 
 public class ServiceBusConsumer extends DefaultConsumer {
 
+    private static final String SERVICE_BUS_RECEIVED_MESSAGE = "ServiceBusReceivedMessage";
```

Comment:
@davsclaus for now, if it is a constant (public static final field) defined in the constant class annotated with `@Metadata`, it will be considered as an header. If we need to improve it more please let me know.

Reply:
As far as I understand, in this case, I guess it will not be annotated and potentially not public but private package so it won't be a problem